src/meta/ActiveHostsMan.cpp

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/ActiveHostsMan.h"
+#include "meta/MetaServiceUtils.h"
+#include "meta/processors/BaseProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+ActiveHostsMan::ActiveHostsMan(int32_t intervalSeconds, int32_t expiredSeconds,
+                               kvstore::KVStore* kv)
+    : intervalSeconds_(intervalSeconds)
+    , expirationInSeconds_(expiredSeconds) {
+    if (kv != nullptr) {
```

Comment:
Do you allow to pass in an empty kv?

Reply:
some ut which only use getActiveHosts pass in an empty kv